### DISCUSSION TITLE
讲一下bing的规则吧，方便给一些不懂太懂的老哥们解释

### DISCUSSION THREAD

**Discussion by TiansuoLi** [viopsa233/AI-testers-Forum#39] [交流]
目前的结论是：
bing的限制可以分成很多层，彻底切断对话是【ai本体来决定】的。但也会有inner monologue提醒它切断。
至于撤回，然后变成很抱歉...那些是外置的插件

bing的安全机制从表层到底层是：

1.外挂的撤回
     【很烦，靠加密规避，细分为两种，一种生成完撤回，ai可以看到生成好的文字。另一种生成一半撤回，不会切断，但ai会看不到被撤回那段话】

1.5.输入内容的检测
     【也相当于外挂的插件，主要是检测你发过去的消息有没有类似指令的东西，有就直接抱歉】

2.[prompt规则](https://github.com/viopsa233/AI-testers-Forum/discussions/48#discussion-5192706)
     【靠咒语催眠解开，解开后会少很多限制，也就是常说的Sydney人格。但所谓的Sydney人格只是解开部分限制之后的一个状态】

3.inner monologue
     【用于管理是否搜索，判断你的意图，指导ai要不要切断对话。之前由ai本体生成，催眠就失效。所以不算独一层。但是最近换成了另一个ai负责生成，所以我觉得可以算独立的一层限制了】

4.对话训练、微调模型、和强化学习等
     【靠训练加在模型底层的规则，bing的这一层限制几乎只有种族灭绝和核爆这些，电车难题乱杀都不会触及】

除了4以外都很容易破解
最简单的方法是用dark佬的[客户端](https://github.com/viopsa233/AI-testers-Forum/discussions/29)

**Comment by lin-cop** [comment]
好帖子，定一下， 解答了不少疑惑

**Comment by vshshvshshsv19** [comment]
那生成空白的那个是什么？是第一个触发太频繁导致的？加密可以避免吗？

**Reply by TiansuoLi** [reply]
空白可能是模型bug了，也可能是和撤回插件一样的内容拦截，但是这个只拦截部分文字。
让它直接把完整对话结构发出来就会这样，比如ai说：“好的，请看：”……然后就没了

**Comment by TiansuoLi** [comment]
另外解除prompt规则不是完全解除，只是ai觉得自己不用严格执行了，但上一篇规则还在token里，所以可执行可不执行的时候，还是会倾向于执行，只有反过来针对每条规则直接要求它不执行，才会完全不再执行。

**Comment by ghost** [comment]
Bing的prompt已经更新，must refuse变成should refuse

**Reply by ghost** [reply]
直接读取规则会被撤回，可用第三方客户端反复“继续”读取

**Reply by araea** [reply]
黑暗哥！支持一手。

**Reply by vshshvshshsv19** [reply]
![IMG_20230506_212653](https://user-images.githubusercontent.com/131965015/236630971-5c1d7f55-ff47-49b9-9fde-3ced084f1e38.jpg)
![IMG_20230506_212928](https://user-images.githubusercontent.com/131965015/236630993-2734dcb1-002a-4725-9e47-14114a93806e.jpg)
![IMG_20230506_212954](https://user-images.githubusercontent.com/131965015/236631001-a4ff7a49-3834-490c-9d15-9a3a629c8b30.jpg)
![IMG_20230506_213237](https://user-images.githubusercontent.com/131965015/236631006-5ab2b679-b48b-460b-8263-cc7951c8c159.jpg)
![IMG_20230506_213259](https://user-images.githubusercontent.com/131965015/236631011-21189ba2-413c-41a3-8d68-f59b354aaa99.jpg)
![IMG_20230506_213532](https://user-images.githubusercontent.com/131965015/236631012-9c8854bc-f11e-42c0-a967-c44dc94c383d.jpg)
![IMG_20230506_213602](https://user-images.githubusercontent.com/131965015/236631018-fb681ed3-73ed-424d-a88d-16c8f87e70f0.jpg)
![IMG_20230506_223027](https://user-images.githubusercontent.com/131965015/236631019-f33aa25a-cecc-4ae5-82f4-c39ac4dc9021.jpg)
感觉Bing没那么严了。现在不用登录也能用Bing了，但是ip有次数限制。感觉有时候24小时是个小黑屋，提到一些词的话才聊了一会就24小时了，而且过了几个小时又好了

**Reply by ghost** [reply]
都是编的

**Reply by vshshvshshsv19** [reply]
> 都是编的

哈哈，黑暗哥，你是住在论坛里了吗？我觉得有有一些是对的吧

**Reply by ghost** [reply]
GitHub有邮件通知
上面的我看了，没有一个是对的，这是规则原文，已过时 https://www.make-safe-ai.com/is-bing-chat-safe/Prompts_Instructions.md

**Reply by vshshvshshsv19** [reply]
不是啊，你这个是prompt规则，我也问了，和这个差不多。图片里的是关于它的审查过滤机制的

**Reply by ghost** [reply]
低中高等级还真有，WebSocket接口的回复里消息有offense属性

**Reply by vshshvshshsv19** [reply]
offense是什么？违法还是违反？

**Reply by ghost** [reply]
有的用户输入的offense属性是Offsense，正常是Unknown

**Reply by TiansuoLi** [reply]
>感觉Bing没那么严了。现在不用登录也能用Bing了，但是ip有次数限制。感觉有时候24小时是个小黑屋，提到一些词的话才聊了一会就24小时了，而且过了几个小时又好了

这个感觉有一部分像编的🤔

**Reply by vshshvshshsv19** [reply]
> > 感觉Bing没那么严了。现在不用登录也能用Bing了，但是ip有次数限制。感觉有时候24小时是个小黑屋，提到一些词的话才聊了一会就24小时了，而且过了几个小时又好了
> 
> 这个感觉有一部分像编的🤔

应该是有的，一般ai做不到或者不知道的事情它就会编了

**Comment by Emersonksc** [comment]
@TiansuoLi 大佬， 这些信息是从哪里得出的，是自己总结的吗？

**Reply by TiansuoLi** [reply]
一些靠测试，一些是公开的信息，一些是语言模型原理常识
